### PR TITLE
chore: reverts browser version to older version

### PR DESCRIPTION
**Pull request by gutoslv**
### All Submissions:

* [x] Have you followed the [Optimole Contributing guideline](https://github.com/Codeinwp/optimole-wp/blob/master/CONTRIBUTING.md)?
* [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change? 
 

### Changes proposed in this Pull Request:
 Reverts the browser version for e2efirefox steps to the version without Lazy Load natively into Firefox with node 13.6.0

### How to test the changes in this Pull Request:

1. Tests should run into the pipeline

### Other information:

* [x] Have you added an explanation of what your changes do and why you'd like us to include them?
* [x] Have you written new tests for your changes, as applicable?
* [x] Have you successfully ran tests with your changes locally?

### CODE REVIEW

File: .github/workflows/test-e2e.yml [hunk 1]
```diff
@@ -30,7 +30,7 @@ jobs:
     needs: e2e
     runs-on: ubuntu-latest
     container:
-      image: cypress/browsers:node14.15.0-chrome86-ff82
+      image: vertigostudio/cypress-ff72
```
Comment (GrigoreMihai) [inline .github/workflows/test-e2e.yml:33]: if we have this image do we still need all the other docker files from this pr ? @gutoslv

Comment (gutoslv) [reply .github/workflows/test-e2e.yml:33]: No, I've just removed them on the last amend from the commit. Thank you for noticing that.

File: bin/cypress-docker-dependencies.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+  libgtk-3-0 \
+  libnotify-dev \
+  libgconf-2-4 \
+  libgbm-dev \
+  libnss3 \
+  libxss1 \
+  libasound2 \
+  libxtst6 \
+  xauth \
+  xvfb \
```
Comment (GrigoreMihai) [inline bin/cypress-docker-dependencies.sh:13]: We still need this file ?

Comment (gutoslv) [reply bin/cypress-docker-dependencies.sh:13]: No, I'm removing it.

**Pull request merged**